File: Android/bitkyShop/app/src/main/java/cc/bitky/bitkyshop/bean/SubCategory.java
package cc.bitky.bitkyshop.bean;

import cn.bmob.v3.BmobObject;

public class SubCategory extends BmobObject {

  private String mainCategory;
  private String photoUrl;
  public String photoName;
  private String name;

  public String getMainCategory() {
    return mainCategory;
  }

  public void setMainCategory(String mainCategory) {
    this.mainCategory = mainCategory;
  }

  public String getPhotoUrl() {
    return photoUrl;
  }

  public void setPhotoUrl(String photoUrl) {
    this.photoUrl = photoUrl;
  }

  public String getName() {
    return name;
  }

  public void setName(String name) {
    this.name = name;
  }

  public String getPhotoName() {
    return photoName;
  }

  public void setPhotoName(String photoName) {
    this.photoName = photoName;
  }
}

File: Csharp/bitkyShop/bitkyShop/bean/SubCategory.cs
using System.Collections.Generic;
using cn.bmob.io;

namespace bitkyShop.bean
{
    public class SubCategory : BmobTable
    {
        public string mainCategory { set; get; }
        public string photoUrl { set; get; }
        public string photoName { set; get; }
        public string name { set; get; }
        //读字段信息
        public override void readFields(BmobInput input)
        {
            base.readFields(input);
            mainCategory = input.getString("mainCategory");
            photoUrl = input.getString("photoUrl");
            photoName = input.getString("photoName");
            name = input.getString("name");
        }

        //写字段信息
        public override void write(BmobOutput output, bool all)
        {
            base.write(output, all);
            output.Put("mainCategory", mainCategory);
            output.Put("photoUrl", photoUrl);
            output.Put("photoName", photoName);
            output.Put("name", name);
        }
    }
}